discovery/kubernetes/ingress.go

```diff
@@ -176,13 +176,24 @@ func (s *Ingress) buildIngress(ingress *v1beta1.Ingress) *targetgroup.Group {
 	}
 	tg.Labels = ingressLabels(ingress)
 
-	schema := "http"
+	tlsHosts := make(map[string]struct{})
 	if ingress.Spec.TLS != nil {
```

Comment:
very minor thing but this test can be removed since the `range ingress.Spec.TLS` will yield nothing when `ingress.Spec.TLS` is nil.